elearning/overview/private-youtube.rst

```diff
@@ -0,0 +1,45 @@
+=================================
+Make videos “Private” on YouTube
+=================================
```

Comment:
I completely agree. The only reason I wrote this documentation was because a colleague in Belgium (Sebastian, I believe) had assigned me this documentation months before I actually started writing any Pull Requests. When I did begin RST/Pull Requests, I recalled that task, and I wrote it up. But, yeah, I am not sure how it really fits in with the documentation itself. So, I will go ahead and delete it. Thanks! :)